src/uct/sm/cma/cma_iface.c

```diff
@@ -23,7 +29,13 @@ static ucs_config_field_t uct_cma_iface_config_table[] = {
 static ucs_status_t uct_cma_iface_get_address(uct_iface_t *tl_iface,
                                               uct_iface_addr_t *addr)
 {
-    *(pid_t*)addr = getpid();
+    pid_t *iface_addr = (void*)addr;
+
+    *iface_addr = getpid() & ~UCT_CMA_IFACE_ADDR_FLAG_PID_NS;
```

Comment:
need to assert that pid does not collide with the flag
don't see the point of passing potentially invalid pid

Reply:
added assert